.travis.yml

```diff
@@ -10,12 +13,23 @@ before_script:
     --with-jpeg-libraries=$JPEG/lib
     --with-magick=$WITH_MAGICK
   - make -j$JOBS -s 
+  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 script:
   - make -j$JOBS -s -k V=0 VERBOSE=1 check
   - LD_LIBRARY_PATH=$PWD/libvips/.libs
     DYLD_LIBRARY_PATH=$PWD/libvips/.libs
     LD_PRELOAD=$ASAN_DSO
     $PYTHON -m pytest -v test/test-suite
+  - docker build --build-arg VIPS_VERSION -t vips/vips-web .
```

Comment:
Could add the `--squash` flag to remove intermediate layers introduced by the use of multiple RUN directives.

Reply:
Done